docs/readme.md

```diff
@@ -1 +1,124 @@
+
+### Installing
+
+A step by step series of examples that tell you how to get a development env running
+
+First clone or download the files on your machine
+
+Place the files in an easy to access folder on your machine
+
+```
+Desktop
+```
+Open your terminal window and run the following command
+```
+nodemon bin/www
+```
+Open your favorite browser and go to
+```
+localhost:3000
+```
+
+#Table Of Contents
+* [Brief]()
+    * [Audience]()
+    * [Problem]()
+    * [Solution]()
+    * [Unique Value Proposition]()
+    * [Pitch]()
+    
+* [Spec]()
+    * [Feature Definitions]()
+    * [Sitemap]()
+    * [Interface]()
+        * [Information Architecture]()
+        * [Browser Support]()
+        
+    * [Infrastructure]()
+        * [Technical Requirements]()
+        * [Programming Languages]()
+        * [Integrations]()
+        * [Deployment Workflow]()
+        * [Web Host]()
+* [Project Milestones]()
+    * [Initial Pitch]()
+    * [Initial Project Spec]()
+    * [MVP]()
+    * [Pivot]()
+    * [Alpha Release]()
+    * [Beta Release]()
+    * [Public release]()
+
+#Brief
+On this brief I will go through the audience, the problem, the solution, my unique value proposition, and pitch.
+I'll identify the audience and the problem, will come up with a solution using my unique value proposition, and will deliver a pitch.
+##Audience
+The audience for this particular fast food restaurant would be adults between the ages of 18 – 50 years old, who work all day and don’t have much time to cook, they would not spend more than $50 on a meal and, for most part they would be single or live with a partner, not necessarily married, and have no children. 
+##Problem
+The problem that this audience have is time. They have a busy lifestyle, not much time to cook or to be hanging on the phone ordering food. They want good, affordable, and fast food.
+##Solution
+The solution will be a online food ordering system.
+##Unique Value Proposition
+Build a web application that can handle order and payment online and give the customer a approximate 'ready time', so the customer knows when the food will be ready and can take care of their busy lives while they wait for the food.
+##Pitch
+We know you are hungry! We'll take care of the food for you.
+
+#Spec
+
+##Feature Definitions
+Filter GitHub Issues: [Features](https://github.com/piazzaana/chicken-fries/issues)
+##Sitemap
+* Home Page
+* Login Page
+* Registration Page
+* Menu Selection Page
+* Breakfast Menu
+* Lunch Menu
+* Dinner Menu
+* About Page
+* Contact Page
+* Location Page
+##Interface
+###Information Architecture
+###Browser Support
+##Infrastructure
+Subject | Technology
+------------ | -------------
+Database | MongoDB
+Web Server | Express
+View Engine | Pug
+Version Control | Git ([GitHub](https://github.com/piazzaana/chicken-fries))
+###Programming Languages
+* NodeJS v10.5.0
+* MongoDB
+###Integrations
+###Deployment Workflow
```

Comment:
It's time to start thinking about your deployment strategy. See Research I in FSO for some potential options. We'd suggest heroku as a good choice, but you're free to explore some of the other options listed. You don't have to set deployment up for the pivot milestone, but you need to decide which you plan to use.

Reply:
Working on it

Reply:
@markmichon I decide to go with Heroku